gulp-tasks/breakpoints.js

```diff
@@ -0,0 +1,137 @@
+/**
+ * @file
+ * Breakpoints parser. Inspired by https://github.com/jenslind/drupal-breakpoints-scss/
+ * Generates sass variables for sass.
+ *
+ * TODO: Put into separated npm package.
+ */
+'use strict';
+const options = require('../gulp-options');
+const yaml = require('js-yaml');
+const fs = require('fs');
+const themeName = options.theme.name;
+const WARNING = 'THIS IS GENERATED FILE. PLEASE EDIT THEMENAME.breakpoints.yml';
+
+/**
+ * Generate sass map with breakpoints.
+ * @param {*} groups - Breakpoints groups list mapped with their queries.
+ * @param {*} fileContent - parsed Drupal breakpoints.yml
+ * @returns Sass Map with drupal breakpoints.
+ */
+
+function generateMap(groups, fileContent) {
+
+  let scssMap = '';
+
+  scssMap += '// ' + WARNING + '\n';
+
+  scssMap += '$breakpoints-groups: (\n';
+
+  groups.forEach((queries, group) => {
+
+    let groupPostfix;
```

Comment:
I'd recommend "groupSuffix" as variable name : 

```suggestion
    let groupSuffix;
```